examples/consistency_distillation/train_lcm_distill_lora_sdxl.py

```diff
@@ -305,7 +310,7 @@ def parse_args():
     parser.add_argument(
         "--cache_dir",
         type=str,
-        default=None,
+        default="/workspace/cache",
```

Comment:
We don't need to change this.

Reply:
Forgot to remove it after local testing .Reverted it to None and pushed a commit. Sorry for the inconvenience.